docs/guides/social-signin/yandex.mdx

```diff
@@ -0,0 +1,108 @@
+---
+id: yandex
+title: Yandex
+---
+
+## Configuration using the Ory CLI 
+
+Follow these steps to add Yandex as a social sign-in provider to your Ory Cloud project
+using the Ory CLI:
+
+1. [Create a Yandex OAuth2 Application](https://yandex.com/dev/oauth/doc/dg/tasks/register-client.html).
+2. In the created app, set the redirect URI to:
+  
+  ```shell
+  https://<ory-cloud-project-slug>.projects.oryapis.com/self-service/methods/oidc/callback/yandex
+  ```
+
+2. Create a Jsonnet code snippet to map the desired claims to the Ory Identity schema.
+
+  ```json
+  local claims = std.extVar('claims');
+  {
+    identity: {
+      traits: {
+        // Yandex don't provide an email_verified field.
+        //
+        // The email might be empty if the user isn't allowed to an email scope.
+        [if "email" in claims then "email" else null]: claims.email,
+      },
+    },
+  }
+  ```
+
+  :::info
+
+  [Yandex](https://yandex.com/dev/oauth/doc/dg/concepts/about.html) returns an `access_token`
+  but doesn't return an `id_token`. Ory sends requests to [Yandex's API](https://yandex.com/dev/passport/doc/dg/reference/request.html)
+  and adds the user info to `std.extVar('claims')`.
+  
+  :::
+
+```mdx-code-block
+import JsonnetWarning from '../../_common/jsonnetwarning.mdx'
+
+<JsonnetWarning format="Jsonnet code snippets" use="data mapping" />
+```
+
+
+3. Encode the Jsonnet snippet with [Base64](https://www.base64encode.org/) or host it under an URL accessible 
+to Ory Cloud.
+4. Download the Identity Service config from your Ory Cloud project and save it to a file: 
+
+  ```shell
+  ## List all available projects
+  ory list projects
+
+  ## Get config
+  ory get identity-config <project-id> --format yaml > identity-config.yaml
+  ```
+
+5. Add the social sign-in provider configuration to the downloaded config. Add the Jsonnet snippet
+with mappings as a Base64 string or provide an URL to the file. 
+
+  ```yaml
+  selfservice:
+    methods:
+      oidc:
+        enabled: true
+        config:
+          providers:
+            - id: yandex # this is `<provider-id>` in the Authorization callback URL. DO NOT CHANGE IT ONCE SET!
+              provider: yandex
+              client_id: .... # Replace this with the OAuth2 Client ID provided by Yandex app
+              client_secret: .... # Replace this with the OAuth2 Client Secret provided by Yandex app
+              mapper_url: "base64://<YOUR_BASE64_ENCODED_JSONNET_HERE>"
+              # Alternatively, use an URL:
+              # mapper_url: https://storage.googleapis.com/abc-cde-prd/9cac9717f007808bf17f22ce7f4295c739604b183f05ac4afb4
+  ```
+
+6. Update the Ory Cloud Identity Service configuration using the file you worked with: 
+
+  ```shell
+  ory update identity-config <project-id> --file updated_config.yaml
+  ```
+
+## Configuration for Self-hosted Instances
+
+Follow these steps to add Auth0 as a social sign-in provider when self-hosting
```

Comment:
```suggestion
Follow these steps to add Yandex as a social sign-in provider when self-hosting
```

Reply:
also in the other docs

Reply:
Thanks man, will make sure to fix that. I bet there's more... :/